src/components/experiment-tracking/run-plots-modal/run-plots-modal.scss

```diff
@@ -2,17 +2,11 @@
 @import '../../../styles/extends';
 
 .kui-theme--light {
-  --color-bg-plot: #{$color-bg-light-4};
-  --color-button-plot: #{$color-button};
-  --color-button-plot-hovered: #{$color-button-hovered};
-  --color-button-text: #{$color-light};
+  --color-bg-plot: #{$white-400};
```

Comment:
Hey why did we need to remove these css values? Are they not needed anymore?

Reply:
They are not used, I found in a few components defined variables at the top which are not used in the css classes, so I removed them.